web/src/index.css

```diff
@@ -1,3 +1,35 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+@layer base {
+  * { @apply box-border; }
+  
+  body, p, ol, ul, td { @apply font-sans leading-5 m-0 text-gray-800; }
+  
+  a { @apply text-gray-800; }
+  
+  a:hover { @apply bg-green-400; }
+  
+  label { @apply block; }
+  
+  button { @apply m-0; }
+  
+  ul { @apply m-0 p-0; }
+}
```

Comment:
Can you add a comment here stating what we're trying to accomplish by adding these?